pkg/provision/vmware/clone.go

```diff
@@ -12,15 +12,15 @@ import (
 
 	cloudinit "github.com/moshloop/konfigadm/pkg/cloud-init"
 	konfigadm "github.com/moshloop/konfigadm/pkg/types"
-	. "github.com/moshloop/platform-cli/pkg/types" // nolint: golint, stylecheck
+	ptypes "github.com/moshloop/platform-cli/pkg/types"
```

Comment:
A bit ugly, happy for a better approach here. 
There is a directive here for the linter to ignore the dot import warning, but later on in `pkg/provision/vmware/vm.go` this confuses
the linter between the VM type defined there, and the VM type defined
in `pkg/types/types.go`